requirements.txt

```diff
@@ -1,5 +1,6 @@
 mypy==0.971
 pre-commit==2.20.0
+python-dotenv==0.20.0
 python-slugify==6.1.2
-pyyaml==6.0.0
+ruamel.yaml==0.17.21
```

Comment:
I have switched to a different package (that is based on the original pyyaml) as it provides better control of indentation.